client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -225,10 +237,12 @@ const fixedProductModel = {
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
 			couponCode: couponCodeField,
-			fixedDiscount: {
-				...fixedDiscountField,
-				labelText: translate( 'Product Discount', { context: 'noun' } )
-			},
+			fixedDiscount: (
+				<CurrencyField
+					labelText={ translate( 'Product Discount', { context: 'noun' } ) }
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.